### PR TITLE
DAC6-2731 : Added Select Address page

### PR DESCRIPTION
**Pull request by AashuChauhanHmrc**

### CODE REVIEW

File: app/models/IndSelectAddress.scala
Comment (BriWak) [inline app/models/IndSelectAddress.scala:1]: This file isn't needed or used and should be removed.

File: test/models/IndSelectAddressSpec.scala
Comment (BriWak) [inline test/models/IndSelectAddressSpec.scala:1]: this spec can be removed as well

File: test/forms/IndSelectAddressFormProviderSpec.scala [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import forms.behaviours.OptionFieldBehaviours
+import models.IndSelectAddress
```
Comment (BriWak) [inline test/forms/IndSelectAddressFormProviderSpec.scala:20]: remove this import as well

**Review by BriWak — CHANGES_REQUESTED** [review]

File: test/models/IndSelectAddressSpec.scala
Comment (AashuChauhanHmrc) [reply test/models/IndSelectAddressSpec.scala:1]: resolved comments

**Review by BriWak — DISMISSED** [review]

**Review by BriWak — APPROVED** [review]

**Pull request merged**